fenix/app/src/main/java/org/mozilla/fenix/share/SaveToPDFMiddleware.kt

```diff
@@ -66,6 +66,20 @@ class SaveToPDFMiddleware(
 
                 postTelemetryFailed(ctx.state.findTab(action.tabId), action.throwable)
             }
+
+            is EngineAction.PrintContentAction -> {
+                next(action)
+                // Reserved for telemetry in bug 1837517
+            }
+            is EngineAction.PrintContentCompletedAction -> {
+                // No-op, reserved for telemetry in bug 1837517
+            }
+            is EngineAction.PrintContentExceptionAction -> {
+                // ToDo: Update after https://github.com/mozilla-mobile/firefox-android/pull/2537 lands
```

Comment:
Will file a bug for this if the toast to snackbar change is not updated in this patch. Holding a little bit to see if the timeline works out with the other bug to change to snackbars in this patch.

Reply:
Will follow up with [bug 1840894](https://bugzilla.mozilla.org/show_bug.cgi?id=1840894) to change to snakbars.